.github/workflows/mirror.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   MIRROR_URL: "git@github.com:EpitechPromo2026/G-EIP-700-NAN-7-1-eip-lucas.hauszler.git"
+  BINARY_NAME: "EngineSquared"
```

Comment:
What is the purpose of this line ? If it is to see if the target exists, the name should be `EngineSquared.lib` and the constant should be called `LIB_NAME` or `LIBRARY_NAME`

Reply:
please look at the fixes I made